jib-cli/build.gradle

```diff
@@ -0,0 +1,35 @@
+plugins {
+  id "application"
+}
+
+ext {
+    cliMainClass = "com.google.cloud.tools.jib.cli.Cram"
+}
+
+// use `run` to build and run the app
+// use `installDist` or `distZip` to create an installable application
+application {
+  applicationName = "jib"
+  mainClassName = cliMainClass
+}
+
+dependencies {
+  implementation project(':jib-core')
+  // since jib core doesn't export dependencies to a compile scope
+  // (they are "runtime"), just grab them manually. This means we don't have
+  // to synchronize dependencies between the two projects -- we don't
+  // want to use the sourceProject helper because it does things we don't want
+  // for an unpublished library.
+  implementation project(':jib-core').configurations.implementation.dependencies
+  implementation "com.google.guava:guava:${dependencyVersions.GUAVA}"
+  implementation "info.picocli:picocli:${dependencyVersions.PICOCLI}"
```

Comment:
I'm curious how all these work out as a CLI tool. Does it not require a JRE? Is it a single executable binary that embeds all transitive dependency JARs? Or, does it cache dependencies under `~/.m2`?

Reply:
Picocli is a nice library for processing command-line arguments.  It also supports more sophisticated command-groups (e.g., `jib build ...` vs `jib jar ...` or `jib inspect ...`), though this code isn't taking advantage of these command-groups yet.

Reply:
Hmm... I don't think I got the answers for my questions, but I think I'll just try this branch myself and see what `./gradlew installDist` generates and how exactly it works on my machine.